packages/block-editor/src/components/block-popover/index.js

```diff
@@ -88,6 +94,29 @@ function BlockPopover(
 
 		return {
 			getBoundingClientRect() {
+				// The zoom out view has a verical block toolbar that should always
+				// be on the edge of the canvas. This condition chnages the anchor
+				// of the toolbar to the section instead of the block to cover for blocks
+				// that are not full width.
```

Comment:
```suggestion
				// The zoom out view has a vertical block toolbar that should always
				// be on the edge of the canvas. This condition changes the anchor
				// of the toolbar to the section instead of the block to handle blocks
				// that are not full width.
```